source/enemies/asteroid.js

```diff
@@ -0,0 +1,73 @@
+const { EnemyTemplate } = require("../classes/index.js");
+const { dealDamage, changeStagger } = require("../util/combatantUtil.js");
+const { selectRandomFoe, selectNone } = require("../shared/actionComponents.js");
+const { getEmoji } = require("../util/elementUtil.js");
+
+const PATTERN = {
+  "Fragment": "random", // can keep fragmenting before randomly exploding on a coin flip
+  "Bolide Burst": "random"
+}
+function asteroidPattern(actionName) {
+  return PATTERN[actionName]
+}
+
+module.exports = new EnemyTemplate("Asteroid",
+  "Earth",
+  85,
+  10,
+  "2",
+  0,
+  "Fragment",
+  false
+).addAction({
+  name: "Fragment",
+  element: "Earth",
+  description: `Inflict ${getEmoji("Earth")} damage to delver, and loses some health`,
+  priority: 0,
+  effect: (targets, user, isCrit, adventure) => {
+    let damage = user.getPower()+30;
+    let recoilDmg = 20;
+    if (isCrit) {
+      damage *= 2;
+    }
+    changeStagger(targets, "elementMatchFoe");
+    return `${dealDamage(targets, user, damage, false, user.element, adventure) + dealDamage([user], user, recoilDmg, false, "Untyped", adventure)}`;
```

Comment:
```suggestion
    return `${dealDamage(targets, user, damage, false, user.element, adventure)} ${dealDamage([user], user, recoilDmg, false, "Untyped", adventure)}`;
```
A template string with only a dynamic value is silly. Instead of the following:
```
`${value}`
```
we should just use `value`.

In the suggestion, I'm using the template string to place the white space joining character. The `+` operator doesn't do that automatically, and is considered a less preferred way of constructing strings because it's less type-safe.